lib/collective.lib.js

```diff
@@ -240,9 +240,13 @@ export const isIndividualAccount = account => ['USER', 'INDIVIDUAL'].includes(ac
 /* Checks whether an account supports grants */
 export const accountSupportsGrants = (account, host) => {
   const accountHasGrantFlag = get(account, 'settings.expenseTypes.hasGrant');
+  const hostHasGrantFlag = get(host, 'settings.expenseTypes.hasGrant');
   if (isBoolean(accountHasGrantFlag)) {
     // If the account's feature flag is set, use it as the main source of truth
     return accountHasGrantFlag;
+  } else if (isBoolean(hostHasGrantFlag)) {
+    // If the host has configured grants to be opt-in
+    return hostHasGrantFlag;
   } else if (get(host, 'settings.disableGrantsByDefault')) {
     // If the host has configured grants to be opt-in, the previous flag (`hasGrant`) **must** be set
```

Comment:
We'll need to review that part in the context of https://github.com/opencollective/opencollective/issues/5395, but no need for you to iterate on it @danielsum